src/biip/gs1/__init__.py

```diff
@@ -75,6 +75,7 @@
 from biip.gs1._symbology import GS1Symbology  # isort:skip  # noqa: E402
 from biip.gs1._application_identifiers import (  # isort:skip  # noqa: E402
     GS1ApplicationIdentifier,
+    _GS1_APPLICATION_IDENTIFIERS,
```

Comment:
Instead of dragging the private data structure via this file, which is mostly for exposing the public interface of the library, import it directly from `biip.gs1._application_identifiers` in `_parser.py`.